spec/src/main/asciidoc/chapters/language/statements.adoc

```diff
@@ -19,22 +21,23 @@ The `from` clause, with syntax given by `from_clause`, specifies an _entity name
 
 NOTE: The syntax of the `update` statement is irregular, with the `from` keyword implied. That is, the syntax _should_ be `update from Entity`, but for historical reasons it is simply `update Entity`.
 
-The `from` clause is optional in `select` statements. When it is missing, the queried entity is determined by the return type of the repository method, or, if the return type is not an entity type, by the primary entity type of the repository.
+The `from` clause is optional in `select` statements. When it is missing, the queried entity is determined by some defaulting rule defined externally to this specification, for example:
 
-For example, this repository method declaration:
+1. a result type might be supplied via an API cainvocationll (for example, via `createQuery()` in Jakarta Persistence), or
```

Comment:
Not sure if you meant call or invocation here.  I'll guess that the latter was typed on top of the former and was probably the intent. I'm fine with either wording.
```suggestion
1. a result type might be supplied via an API invocation (for example, via `createQuery()` in Jakarta Persistence), or
```